applications/asset_tracker_v2/src/cloud/nrf_cloud_integration.c

```diff
@@ -228,24 +222,20 @@ static void nrf_cloud_event_handler(const struct nrf_cloud_evt *evt)
 		cloud_wrap_evt.type = CLOUD_WRAP_EVT_FOTA_ERROR;
 		notify = true;
 		break;
-	case NRF_CLOUD_EVT_RX_DATA:
-		LOG_DBG("NRF_CLOUD_EVT_RX_DATA");
-
-		/* Filter out responses to cellular position requests. The application does not care
-		 * about getting its location back after neighbor cell measurements have been
-		 * sent to cloud.
-		 */
-		err = cell_pos_response_filter((char *)evt->data.ptr, evt->data.len);
-		if (err) {
-			LOG_DBG("Cellular position response received, aborting");
-			return;
-		}
-
+	case NRF_CLOUD_EVT_RX_DATA_GENERAL:
+		LOG_DBG("NRF_CLOUD_EVT_RX_DATA_GENERAL");
+		break;
+	case NRF_CLOUD_EVT_RX_DATA_SHADOW:
+		LOG_DBG("NRF_CLOUD_EVT_RX_DATA_SHADOW");
+		/* Configuration data is contained in the shadow events */
 		cloud_wrap_evt.type = CLOUD_WRAP_EVT_DATA_RECEIVED;
 		cloud_wrap_evt.data.buf = (char *)evt->data.ptr;
 		cloud_wrap_evt.data.len = evt->data.len;
 		notify = true;
 		break;
+	case NRF_CLOUD_EVT_RX_DATA_CELL_POS:
+		LOG_DBG("NRF_CLOUD_EVT_RX_DATA_CELL_POS");
```

Comment:
I have another PR where I'm reading cell pos results in ATv2. I guess I can find them still at `evt->data.ptr`, right?

Reply:
correct